draft-wang-ppm-differential-privacy.md

```diff
@@ -312,13 +313,12 @@ both Clients and Aggregators may participate in generating the noise.
 The primary goal of this document is to specify how DP policies are implemented
 in DAP. It does so in the following stages:
 
-1. {{overview}} describes the notion(s) of DP that are compatible with DAP and
-   provides a systematization of applicable DP policies from the literature.
+1. {{overview}} describes the notion(s) of DP that are compatible with DAP.
    Security is defined in a few different "trust models" in which we assume
    that some fraction of the parties execute the protocol honestly. Of course
    in reality, whether such assumptions hold is usually outside of our control.
    Thus our goal is to design DP policies that still provide some degree of
-   protection in more pessimistic trust models.
+   protection in more pessimistic trust models. (We call this "hedging".)
 
 1. {{mechanisms}} specifies various mechanisms required for building DP
```

Comment:
Not related to this PR: but do we want to have a bullet point to mention “use cases” where we show how different DP policies can be used on?

Reply:
Good call. I'll merge this, then do you wanna follow up with a PR?